index.html

```diff
@@ -1,64 +1,279 @@
+          <h2 class="new">Multi-Post Stories Gain+Glory</h2>
+          <ul class="list p-lang">
             <li>
-              <a href="http://www.linkedin.com/in/samuel-ntow-kyere-5036741b4"
-                ><i class="fa-brands fa-linkedin"></i
-              ></a>
+              <button class="btn-btn" type="button">Ruby on rails</button>
             </li>
+            <li><button class="btn-btn" type="button">Css</button></li>
+            <li><button class="btn-btn" type="button">Javascript</button></li>
+            <li><button class="btn-btn" type="button">Html</button></li>
+          </ul>
+          <button class="btn view" type="button">See Project</button>
+        </div>
+      </div>
+      <div class="card-flex">
+        <div class="description">
+          <h2 class="new">Multi-Post Stories Gain+Glory</h2>
+          <ul class="list p-lang">
             <li>
-              <a href="https://twitter.com/brown_admirer"
-                ><i class="fa-brands fa-twitter"></i
-              ></a>
+              <button class="btn-btn" type="button">Ruby on rails</button>
             </li>
+            <li><button class="btn-btn" type="button">Css</button></li>
+            <li><button class="btn-btn" type="button">Javascript</button></li>
+            <li><button class="btn-btn" type="button">Html</button></li>
           </ul>
+          <button class="btn view" type="button">See Project</button>
         </div>
-        <div class="headline-button">
-          <img src="images/Action.svg" />
+      </div>
+      <div class="card-flex">
+        <div class="description">
+          <h2 class="new">Multi-Post Stories Gain+Glory</h2>
+          <ul class="list p-lang">
+            <li>
+              <button class="btn-btn" type="button">Ruby on rails</button>
+            </li>
+            <li><button class="btn-btn" type="button">Css</button></li>
+            <li><button class="btn-btn" type="button">Javascript</button></li>
+            <li><button class="btn-btn" type="button">Html</button></li>
+          </ul>
+          <button class="btn view" type="button">See Project</button>
         </div>
       </div>
+    </div>
+  </section>
+
+  <!-- recent About-me section -->
+  <section id="about-me">
+    <div class="about">
+      <div class="about-li">
+        <a>
+          <h1 class="ab-head">About me</h1>
+        </a>
+
+        <a><img class="side1" src="images/side.svg" alt="side-image" /></a>
+      </div>
+
+      <p class="ab-text">
+        I can help you build a product , feature or website Look through some
+        of my work and experience! If you like what you see and have a project
+        you need coded, don’t hesitate to contact me.
+      </p>
+
+      <div class="about-li">
+        <img class="" src="images/Frame-1.svg" alt="side-image" />
+
+        <button class="btn ab-btn-btn" type="button">Get my Resume</button>
+      </div>
+    </div>
+
+    <section id="abt-me-items">
```

Comment:
- [ ] like the previous reviewer noted, the about me items should be inside the about me `section tag` not on their own personal section tag. They all make up the skills and framework about you and should be together. please fix that.